vm/src/pyobject.rs

```diff
@@ -1118,9 +1118,14 @@ impl PyObject {
     }
 
     #[inline]
-    pub fn payload<T: PyValue>(&self) -> Option<&T> {
+    pub fn payload<T: PyObjectPayload>(&self) -> Option<&T> {
         self.payload.as_any().downcast_ref()
     }
+
+    #[inline]
+    pub fn payload_is<T: PyObjectPayload>(&self) -> bool {
```

Comment:
nit: would prefer this just be called `is`

(I'm thinking once there's a `downcast` or `as` on `PyRef`, we'll move away from using `payload`, so calling this `is` would provide a consistent api)

Reply:
Oh, bummer, either is fine for now then